lib/http/client.rb

```diff
@@ -24,7 +24,12 @@ def initialize(default_options = {})
       @state = :clean
     end
 
-    # Make an HTTP request
+    # Make an HTTP request.
+    # See HTTP::Options.defined_options for list of option keys.
+    # @see https://ruby-doc.org/stdlib-2.5.1/libdoc/openssl/rdoc/OpenSSL/SSL/SSLContext.html OpenSSL::SSL::SSLContext for more information on :ssl options.
+    # @param [Symbol] verb The HTTP method.
```

Comment:
Please delegate this documentation to `#build_request`:
```suggestion
    # @param verb (see #build_request)
```